src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -58,6 +58,18 @@ public void test_nameContainsKeywords_returnsTrue() {
         assertTrue(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
     }
 
+    @Test
+    public void test_phoneContainKeywords_returns() {
+        // Zero keywords
+        NameContainsKeywordsPredicate predicate = new NameContainsKeywordsPredicate(Collections.emptyList());
+        assertFalse(predicate.test(new PersonBuilder().withPhone("12345").build()));
```

Comment:
It looks good! One note is that the phone number in this test case is not 8 digits long like specified in the feature specification document. 

I actually added the constrain of 8 digit long phone numbers in PR #44 as well as changed all the sample Person objects to have 8 digit phone numbers. So this test case might fail once that is merged.